src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts

```diff
@@ -277,6 +279,10 @@ export class CoursePlayerComponent implements OnInit, OnDestroy {
       queryParams: { 'contentId': content.id },
       relativeTo: this.activatedRoute
     };
+    const contentDet = this.findContentById(content.id);
```

Comment:
what is the variable name ? can you have a name more descriptive name

Reply:
changed the name to playContentDetail